doc/RELEASE-0.8.md

```diff
@@ -0,0 +1,49 @@
+
+# Release v0.8
+
+
+We are pleased to announce the release of Screed v0.8. Screed is a database
+engine capable of storing and retrieving short-read sequence data. Screed is
+designed to be fast and adaptable to different sequence file formats. This 
+release of Screed we consider to be stable and complete.
+
+This version of Screed contains developer documentation for contributing to the
+Screed project and a code of conduct for interacting with other contributors
+and project maintainers. 
+
+## Features and Major Changes:
+ - bz2 compressed file streaming
+ - 'accuracy' attribute has been changed to 'quality'
+ - versioneer version string generator has been added
+ - dooxygen documentation engine has been added
+ - code of conduct and developer documentation has been added
+ - detailed changelog has been added
+
+## Closed Issues:
+    * [Screed gzip/bz2 file sniffing]
```

Comment:
The extra indentation is causing this to render incorrectly. Also, why switch to '*' from '-' above?